rustup-init.ps1

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/pwsh
+
+# This is just a little script that can be downloaded from the internet to
+# install rustup. It just does platform detection, downloads the installer and
+# runs it.
+
+#XXX: If you change anything here, please make the same changes in setup_mode.rs
```

Comment:
This should say setup_mode.rs and rustup-init.sh; the same comment in both setup_mode.rs and rustup-init.sh needs to be updated to point also to this new file: the command line options should be kept in sync in all three places in the intent of the comment. It could perhaps be clarified too.

Reply:
Good call, I updated this comment and included the Rust source and the Shell script in this change, lest they be forgotten later.